VEHICLE/N_0x8aa9180de2fedd45.md

```diff
@@ -1,15 +1,19 @@
 ---
 ns: VEHICLE
 ---
-## _0x8AA9180DE2FEDD45
+## EnableAircraftObstacleAvoidance
 
 ```c
 // 0x8AA9180DE2FEDD45 0xAE040377
-void _0x8AA9180DE2FEDD45(Vehicle vehicle, BOOL p1);
+void _0x8AA9180DE2FEDD45(Vehicle vehicle, BOOL enableobstacleavoidance);
 ```
 
 
+Will disable a plane or a helicopter's need to swerve around object in its heightmap.  Will ensure plane flys directly to it's destination or die trying! This native does NOT need to be called every frame, but instead, just called once on the vehicle (NOT THE PED) you're trying to disable avoidance for!
```

Comment:
As you mentioned in your [post](https://forum.cfx.re/t/pedbehavior-how-do-i-get-the-ped-to-kamikaze-aka-how-can-i-get-the-ped-to-not-care-about-their-own-demise/5102765), perhaps you should mention that this native is meant to be used for AI behavior purposes.